examples/editor-core/overlays/app.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
```

Comment:
Nit: We should change the directory name from editor-core to editable-layers

Reply:
Unrelated comment: I recommend creating a `modules/react` `@deck.gl-community/react` and move all general react components there. These overlays have no dependencies on editable-layers, and the same is true for a number of the components in react-graph-layers.

Reply:
We want to eventually sunset the graph and editor specific react modules if at all possible, so stripping out common stuff makes a ton of sense.

Reply:
@ibgreen What would you think of raising the nested examples up a level, so we have something like:

- examples/template
- examples/graph-layers
- examples/editable-layers-basic
- examples/editable-layers-advanced
- examples/editable-layers-overlays
- ...

It doesn't seem like deck.gl subdivides examples by submodule, we may want other folder divisions for the website, and I think it's slightly easier for users to explore a flat list.

@weberjavi I'll merge the PR after CI passes, no need to address any of the above in this PR. Thanks again!